Formula/mikutter.rb

```diff
@@ -0,0 +1,378 @@
+    sha256 "7e0a43c527af605dc1fc3ef8416d2641b400078d8a22e30981ac47d81af290ab"
+  end
+  resource "pluggaloid" do
+    url "https://rubygems.org/gems/pluggaloid-1.1.1.gem"
+    sha256 "f9279fad38d0bf4e20ee70e30882c6cb7916bc764bf72b2f955f0ac0ff0a3a5d"
+  end
+  resource "power_assert" do
+    url "https://rubygems.org/gems/power_assert-0.3.1.gem"
+    sha256 "f89ba9cc1fa9d684b8f9f1221e94a8472a89355b5a6cb40d99fc863cbb4bec3a"
+  end
+  resource "ruby-hmac" do
+    url "https://rubygems.org/gems/ruby-hmac-0.4.0.gem"
+    sha256 "a4245ecf2cfb2036975b63dc37d41426727d8449617ff45daf0b3be402a9fe07"
+  end
+  resource "test-unit" do
+    url "https://rubygems.org/gems/test-unit-3.2.1.gem"
+    sha256 "50599b29ba3edf1aa924bca6341989fdd27e5c72d4d5bb66dfc65fc5f7cbcfb0"
+  end
+  resource "totoridipjp" do
+    url "https://rubygems.org/gems/totoridipjp-0.1.0.gem"
+    sha256 "93d1245c5273971c855b506a7a913d23d6f524e9d7d4494127ae1bc6174c910d"
+  end
+  resource "unf_ext" do
+    url "https://rubygems.org/gems/unf_ext-0.0.7.2.gem"
+    sha256 "e8fa13d09880f8d06d30a86f929a38ba0af6abe61272927a49e6796aee1aa111"
+  end
+  resource "unf" do
+    url "https://rubygems.org/gems/unf-0.1.4.gem"
+    sha256 "4999517a531f2a955750f8831941891f6158498ec9b6cb1c81ce89388e63022e"
+  end
+  resource "twitter-text" do
+    url "https://rubygems.org/gems/twitter-text-1.14.0.gem"
+    sha256 "8c60d5be2dcfe19bb97d2f2549296cf5d4f52199de85ea7bb66e1196193cd294"
+  end
+  resource "typed-array" do
+    url "https://rubygems.org/gems/typed-array-0.1.2.gem"
+    sha256 "891fa1de2cdccad5f9e03936569c3c15d413d8c6658e2edfe439d9386d169b62"
+  end
+  resource "watch" do
+    url "https://rubygems.org/gems/watch-0.1.0.gem"
+    sha256 "1d3e767cb917f226cb970ac0e39c9ee613f9082a390598bf94be516bbd79e409"
+  end
+  resource "webmock" do
+    url "https://rubygems.org/gems/webmock-1.24.6.gem"
+    sha256 "c516e1b309697af303e647dc2f3c7222b13ef70c1c4c5afb61e64bd595c9740f"
+  end
+
+  def install
+    mkpath libexec/"vendor"
+    resources.each do |r|
+      r.verify_download_integrity(r.fetch)
+      system("gem", "install", r.cached_download, "--no-document",
+             "--install-dir", "#{libexec}/vendor")
+    end
+
+    prefix.install_metafiles
+    rm_rf "vendor"
+    libexec.install Dir["*"]
+
+    if build.with? "terminal-notifier"
+      mkpath libexec/"plugin"
+      (libexec/"plugin/macos_compatibility.rb").write(compatibility_plugin)
+    end
+
+    mkpath bin
+    (bin/"mikutter").write(exec_script)
+
+    mkpath prefix/"mikutter.app/Contents/MacOS"
+    (prefix/"mikutter.app/Contents/MacOS").install_symlink bin/"mikutter"
+
+    mkpath prefix/"mikutter.app/Contents/Resources"
+    system("/usr/bin/sips", "-s", "format", "icns",
+           libexec/"core/skin/data/icon.png",
+           "--out", "#{prefix}/mikutter.app/Contents/Resources/mikutter.icns")
+
+    (prefix/"mikutter.app/Contents/Info.plist").write(info_plist)
+  end
+
+  def exec_script
+    <<-EOS.undent
```

Comment:
You don’t have to depend on `HOMEBREW_PREFIX`; just use the binaries available in the `PATH`.